docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1506,3 +1506,93 @@ For `@QuarkusIntegrationTest` tests that result in launcher the application as a
 This can be used by `QuarkusTestResourceLifecycleManager` that need to launch additional containers that the application will communicate with.
 ====
 
+== Testing Components
+
+In Quarkus, the component model is built on top CDI.
+Therefore, Quarkus provides the `QuarkusComponentTest`, a JUnit extension to ease the testing of components and mocking of their dependencies.
+
+Let's have a component `Foo`:
+
+[source, java]
+----
+package org.acme;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+@ApplicationScoped <1>
+public class Foo {
+
+    @Inject
+    Charlie charlie; <2>
+
+    public String ping() {
+        return charlie.ping();
+    }
+}
+----
+<1> `Foo` is an `@ApplicationScoped` CDI bean.
+<2> `Foo` depends on `Charlie` which declares a method `ping()`.
+
+Then a component test could look like:
+
+[source, java]
+----
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.mockito.Mockito;
+import org.acme.Charlie;
+import org.acme.Foo;
+
+public class FooTest {
+
+    @RegisterExtension <1>
+    static final QuarkusComponentTest test = new QuarkusComponentTest(Foo.class);
+
+    @Inject
+    Foo foo; <2>
+
+    // Inject a mock that is created automatically...
+    @Inject
+    Charlie charlie; <3>
+
+    @Test
+    public void testPing() {
+        Mockito.when(charlie.ping()).thenReturn("OK"); <4>
+        assertEquals("OK", foo());
```

Comment:
```suggestion
        assertEquals("OK", foo.ping());
```

Reply:
Thanks Many! :facepalm:

Reply:
No worries, thanks for the great addition!